src/scss/style.scss

```diff
@@ -14,22 +14,207 @@
   background-image: url("../assets/images/icons/zoom-in-hover.svg");
 }
 
-.lr-tooltip-text {
-  max-width:100px;
-  padding:5px;
-  min-height:10px;
-  background:#000;
-  visibility: hidden;
-  color: #fff;
+/* Button primary (White background) style rules --- Start */
+.sgt-btn-one {
+  box-sizing: border-box;
+  border-radius: 4px;
+  height: 40px;
+  min-width: 64px;
+  padding-left: 16px;
+  padding-right: 16px;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 14px;
+  line-height: 20px;
+  text-align: center;
+  color: #ffffff;
+}
+.sgt-btn-one:focus {
+  outline: 0;
+}
+.sgt-btn-one:disabled {
+  background: #b7b7b7;
+}
+.sgt-btn-one-information {
+  background: #00a1dd;
+}
+.sgt-btn-one-information:hover {
+  background: #00b2ff;
+}
+.sgt-btn-one-information:active {
+  background: #0078d6;
+}
+.sgt-btn-one-success {
+  background: #6ebe49;
+}
+.sgt-btn-one-success:hover {
+  background: #8acb6b;
+}
+.sgt-btn-one-success:active {
+  background: #4b813d;
+}
+.sgt-btn-one-warning {
+  background: #f7a528;
+}
+.sgt-btn-one-warning:hover {
+  background: #ffb744;
+}
+.sgt-btn-one-warning:active {
+  background: #d98a13;
+}
+.sgt-btn-one-critical {
+  background: #dc1f2e;
+}
+.sgt-btn-one-critical:hover {
+  background: #ea3947;
+}
+.sgt-btn-one-critical:active {
+  background: #af131f;
+}
+/* Button primary (White background) style rules --- End */
+
+/* Button secondary (White background) style rules --- Start */
+.sgt-btn-two {
+  background: #ffffff;
+  box-sizing: border-box;
+  border-radius: 4px;
+  height: 40px;
+  min-width: 64px;
+  padding-left: 16px;
+  padding-right: 16px;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 14px;
+  line-height: 20px;
   text-align: center;
-  border-radius: 6px;
-  /* Position the tooltip */
-  position: absolute;
-  z-index: 1;
-  word-wrap:break-word;
+  color: #000000;
+  border: 1px solid #000000;
+}
+.sgt-btn-two:focus {
+  outline: 0;
+}
+.sgt-btn-two:disabled {
+  color: #b7b7b7;
```

Comment:
1. are all these classes used across project since added in style.scss
2. the colors assigned are # code, are these colors not a constant like (csmprimary, etc).

Reply:
Yes, those are common classes that can be used across your project. I have now changed the hex color codes to the SASS variables.